.gitmodules

```diff
@@ -2,3 +2,6 @@
 	path = ethcore/res/ethereum/tests
 	url = https://github.com/ethereum/tests.git
 	branch = develop
+[submodule "nsis/win-build"]
```

Comment:
NSIS is preinstalled with appveyor, but without plugin
submodule is just a way to install plugin